src/main/resources/sponsorblock/host/values/strings.xml

```diff
@@ -35,6 +35,9 @@
     <string name="sb_guidelines_popup_open">Show me</string>
 
     <string name="sb_general">General</string>
+    <string name="sb_toast_on_connection_error_title">Show toast if SB not available</string>
```

Comment:
```suggestion
    <string name="sb_toast_on_connection_error_title">Show toast if SponsorBlock is not available</string>
```

Reply:
We should avoid arbitrary abbreviations anywhere where possible if not explicitly stated otherwise. The reason is that services are not necessarily represented by those abbreviations in the same way ReVanced is never represented as "RV" unless explicitly specified. SponsorBlock is never referred to as anything besides "SponsorBlock" officially. Thus I think it would only be correct to refer to it as "SponsorBlock," too. The same applies to ReturnYouTubeDislike, btw. Is there a way to avoid clipping?